packages/loopring_v3/contracts/iface/ExchangeData.sol

```diff
@@ -38,20 +40,6 @@ library ExchangeData
         TRANSFER
```

Comment:
remove ORDER_CANCELLATION

Reply:
should we rename RING_SETTLEMENT =>SETTLEMENT?

Reply:
I would keep `ORDER_CANCELLATION` because `TRANSFER` is behind it. Changing it would change which number is used for which circuit type compared to V3.0 which I think could be very confusing. So to be safe I kept in the list of types because why not, it's just a circuit type we won't support any more.